app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'comment'
+
+class Post < ApplicationRecord
+  after_initialize :update_posts_counter
+  has_many :comments
+  belongs_to :author, class_name: 'User'
```

Comment:
- [ ] I think it would be great if you add here the associations between the post and like tables as well so here in post model it also should contain has many likes so each post could have many likes and many comment, great you added many likes but you forget to add many like relation so kindly add it 💪🏻💯♻✨